parlor_test.go

```diff
@@ -4,47 +4,68 @@ import (
 	"testing"
 
 	"github.com/blocky/parlor"
-	"github.com/blocky/parlor/internal/mocks"
 )
 
-type ParlorTestSuite struct {
+type ParlorYesSetupNoTearDown struct {
```

Comment:
Do we want a `ParlorYesSetupYesTearDown` ?

Reply:
It is a good question.  I had it originally... In fact, I had all 4 (all combinations of "XSetupYTearDown") but when I wrote the tests, "Yes...Yes..." and "No...No..." seemed redundant.